runelite-client/src/main/java/net/runelite/client/plugins/dailytaskindicators/DailyTasksPlugin.java

```diff
@@ -139,6 +139,10 @@ public void onGameTick(GameTick event)
 			sendChatMessage("You have pure essence waiting to be collected from Wizard Cromperty.");
 			hasSentEssenceMsg = true;
 		}
+		if (config.showRunes() && canCollectRunes())
+		{
+			sendChatMessage("You have random runes waiting to be collected from Lundail.");
```

Comment:
You should maybe also add boolean that the message was sent in order to not send it every time